lms/envs/common.py

```diff
@@ -1760,6 +1760,9 @@
     'rest_framework',
     'openedx.core.djangoapps.user_api',
 
+    # Team API
+    'openedx.core.djangoapps.team_api',
```

Comment:
The API should live in lms/djangoapps/teams. We only want a single teams django app, and #8168 is introducing the teams django app in LMS since we don't think it will ever be needed by Studio.